Marlin/src/HAL/HAL_STM32/timers.h

```diff
@@ -63,7 +63,7 @@
   #define HAL_TIMER_RATE (F_CPU/2) // frequency of timer peripherals
 
   #ifndef STEP_TIMER
-    #define STEP_TIMER 6
+    #define STEP_TIMER 9  // timers 5 - 8 do not exist on the STM32F40x chips
```

Comment:
Should there be a separation between the values for STM32F4 and STM32F7 here? Currently they will just use the same values.

Reply:
They can both use timer 9 for the STEPPER_TIMER.  Both have timer 9, the capabilities are the same for both devices and I've tested it on the F401.

On the other hand it's OK to separate them.  

Not a big deal either way.